modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -123,15 +126,17 @@ FeatureFloodCount::FeatureFloodCount(const InputParameters & parameters) :
     _maps_size(_single_map_mode ? 1 : _vars.size()),
     _n_procs(_app.n_processors()),
     _entities_visited(_vars.size()), // This map is always sized to the number of variables
+    _feature_counts_per_map(_maps_size),
     _feature_count(0),
+    _max_local_size(0),
     _partial_feature_sets(_maps_size),
-    _feature_sets(_maps_size),
     _feature_maps(_maps_size),
     _pbs(nullptr),
     _element_average_value(parameters.isParamValid("elem_avg_value") ? getPostprocessorValue("elem_avg_value") : _real_zero),
     _halo_ids(_maps_size),
     _compute_boundary_intersecting_volume(getParam<bool>("compute_boundary_intersecting_volume")),
-    _is_elemental(getParam<MooseEnum>("flood_entity_type") == "ELEMENTAL" ? true : false)
+    _is_elemental(getParam<MooseEnum>("flood_entity_type") == "ELEMENTAL" ? true : false),
```

Comment:
Ha - good catch... Funny what some of the code we crap out looks like sometimes.